src/register/RegistrationPage.jsx

```diff
@@ -188,6 +190,7 @@ class RegistrationPage extends React.Component {
       }
     });
     if (finalValidation) {
+      payload.totalRegistrationTime = totalRegistrationTime.toString();
```

Comment:
Why do we need to convert it into a string? Also, can we also do the calculations here once all validations pass?